tests/suites/test_suite_shax.data

```diff
@@ -186,2231 +186,2239 @@ depends_on:MBEDTLS_SELF_TEST:MBEDTLS_SHA512_C
 
 SHA3_384 long #28
-depends_on:MBEDTLS_SHA3_C
+depends_on:PSA_WANT_ALG_SHA3_384
 mbedtls_sha3_multi:MBEDTLS_SHA3_384:"daf3dafeb903e95c40cbb9ae4028ca494ab948721f89282168ef7405a7b3e3c76e72ad93cf449dd5139b6840ae6f3d1573989da2057b6d098ee84a42b3431c63c7e2ee1f3eb7fd8d652f0e2b51590523154bb4cd39c2f1746708eb5d661bbe054d0b44abf0a42e0e9f4d547810304afb999b34fba2a6dbed5b17005daa82f03668b5583ecc535be027bee22ed23a33266550c6342a7153f9d2dc304b66c2da7d80a7d2e0ba0baa89674a5f5581c1e92be3321181b86983223420242e8221a562fdaa5999a2154393c5b874f6040b94927a08bee935d891efe9ab830ee5e8796d478695726cb94f86abd2c81843605ae4919c74758704a5e85945f6f4795e7d27c6032f72013ff6e8805e04c3260a66a892b4dc2c9736e896ecb625e625cdda17e2e8cfa247ef5f43ac25cfff9550de66e4199d164db581800bc5b7efd17bb7c604ff1c082711b31e97efd2d57879eba7ef93603b91cb992f1c3c551465b13dba920ebfae89ec1902a984b44cb7ceb4f11beacda44cbe582989e1a65fcab7947df6de5f9c56c9ba764aab1e6d3a9762cfd0e58b5864d7e5f8b87d53d7e6d5e3979637a5d974f0be02e22eb83fe91c1cc6764d435fc35e692313d70384a253528edce1396dcda94c2ed9d329caada40b6075ea9cc24f9b0b3300b95f6fe7780bec2fa3c260483c8c8397631a166c78562435843f
```

Comment:
Minor (and pre-existing): since the function already depends on this, we don't need to repeat it here. (Doesn't hurt either, so feel free to ignore.)

Reply:
Hmm, github is playing tricks on us. My comment was on SHA-3 invalid param where the function does depend on a specific SHA3 variant. That's how it shows if you go to the files view, click [on this data file](https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/266/files#diff-4ad9d57b88d6828df9a8be918c7c6e09da6d44f21035f2c49eaf57282b63ed57) then "load diff" and ctrl-F invalid. I have no idea why it's showing on a different line in the summary page. Quite annoying.